bin/post_compile

```diff
@@ -0,0 +1,16 @@
+set -e
+buildpack_url=https://github.com/heroku/heroku-buildpack-python
+buildpack_vsn=v197
```

Comment:
Looks like the buildpack was needed like Dan suggested?

Reply:
Yes! I also had to reorder everything and then this thing is what the woman on GitHub suggested, which is a script that, after Python builds, rebuilds it with shared libraries.